Userland/Libraries/LibJS/Runtime/Error.h

```diff
@@ -24,6 +24,7 @@ class Error : public Object {
     virtual ~Error() override = default;
 
     ThrowCompletionOr<void> install_error_cause(Value options);
+    void populate_stack();
```

Comment:
This function should probably be private or even just not exist and the steps done in the constructor.

Reply:
I made it private for now.